plone/app/discussion/browser/moderation.py

```diff
@@ -205,17 +207,24 @@ class PublishComment(BrowserView):
     """
 
     def __call__(self):
+        alsoProvides(self.request, IDisableCSRFProtection)
```

Comment:
I think it would be better to add a CSFR token to the template where the URL to this call is generated instead of disabling the protection.